recipes/py6s/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "py6s" %}
```

Comment:
I guess that the name is `Py6S`, right? don't worry with the lower case conda name because the jinja below takes care of that.